components/SnackBar/SnackBar.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+import PropTypes, { oneOf } from 'prop-types';
+import Colors from '../Colors';
+import Button from '../Button';
+import { Row, Col } from '../Grid';
+import uniqId from '../shared/uniqId';
+
+const a11yFocusTab = `outline: auto;`;
+
+const getBackgroundBySkin = skin => {
+  const color = skin.toUpperCase();
+  const indexColor = skin === 'cobalt' ? '500' : '700';
+  return `
+      background-color: ${Colors[color][indexColor]}
+  `;
+};
+
+const SnackBarDialog = styled.div`
+  align-items: center;
+  border-radius: 4px;
+  color: ${Colors.WHITE};
+  display: flex;
+  justify-content: space-between;
+  min-height: 48px;
+  padding: 13px 16px;
+
+  ${({ skin }) => getBackgroundBySkin(skin)}
+`;
+
+const TextContainer = styled.strong`
+  font-weight: normal;
+  padding-right: 8px;
+
+  :focus {
+    ${a11yFocusTab}
+  }
+`;
+
+const CloseIcon = styled(Button.Icon).attrs({
+  icon: 'close',
+})`
+  color: ${Colors.WHITE};
+
+  :hover,
+  :focus {
+    background-color: transparent;
+    box-shadow: none;
+    color: ${Colors.WHITE};
+    ${a11yFocusTab}
+  }
+`;
+
+const ActionButton = styled(Button)`
+  color: ${Colors.WHITE};
+  font-weight: bold;
+  text-transform: uppercase;
+  white-space: nowrap;
+
+  :hover,
+  :focus {
+    text-decoration: none;
+    color: ${Colors.WHITE};
+  }
+
+  :focus {
+    ${a11yFocusTab}
+  }
+`;
+
+const ActionsSection = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+const DialogBlock = styled.section`
+  bottom: 0;
+  left: 0;
+  position: fixed;
+  width: 100%;
+`;
+
+class SnackBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.snackBarSection = document.createElement('section');
+  }
+
+  componentDidMount() {
+    const { body } = document;
+    body.appendChild(this.snackBarSection);
+
+    this.objectFocus(document.querySelector('.action-button'));
+
+    const { onClose, secondsToClose } = this.props;
+
+    setTimeout(() => {
+      onClose();
+    }, secondsToClose * 1000);
+
+    this.closeOnTime(secondsToClose, onClose);
+  }
+
+  componentWillUnmount() {
+    const { body } = document;
+    body.removeChild(this.snackBarSection);
+  }
+
+  // this event was implemented to simulate unit tests (a11y)
+  handleKeyPress = event => {
+    const { onClose } = this.props;
+    if (event.key === 'Enter') onClose();
+  };
+
+  objectFocus = element => {
+    element.focus();
+  };
+
+  closeOnTime = (secondsToClose, callback) => {
+    setTimeout(() => {
+      callback();
+    }, secondsToClose * 1000);
+  };
+
+  render() {
+    const _id = uniqId('snackbar-dialog-');
+    const {
+      text,
+      onClose,
+      closeButtonAriaLabel,
+      actionTrigger,
+      ...rest
+    } = this.props;
+    return ReactDOM.createPortal(
+      <DialogBlock>
+        <Row>
+          <Col xsmall-offset={1} xsmall={10}>
```

Comment:
This values are wrong, the xsmall have only 4 columns, small 6 columns, medium, large and xlarge have 12 columns.

If you test on a small resolution you will see that the snackbar is at the right corner of the viewport.

![image](https://user-images.githubusercontent.com/13424727/55431419-e647c880-5566-11e9-8741-0b110f667a53.png)